data/sheragi/archaeology missions.txt

```diff
@@ -1170,7 +1170,11 @@ mission "Sheragi Archaeology: The Box 7b"
 			`	"I see. I'll have to warn you though, following the footsteps of a doomed species can only get you so far.`
 				goto end
 			label truth
-			`	"A Sheragi black box? That's an incredible find! Be careful about the information it contains, though. They are a doomed species. Don't you humans have a legend about a cursed box? What was it... Pandora's Box?`
+			`	You produce an image of the cube that you found on your holopad. The curator stares at it, and then slowly turns to you with pity in his eyes.`
+			`	"This box is red," he says gently. "I read a thesis once on the prevalence of color blindness in humans, but-"`
+			`	"No, wait!" you cut him off. "That's a human idiom. 'Black box' is what we use to refer to a device designed to collect and preserve flight data in the event of a catastrophic failure."`
```

Comment:
```suggestion
			`	"No, wait!" You cut him off. "That's a human idiom. 'Black box' is what we use to refer to a device designed to collect and preserve flight data in the event of a catastrophic failure."`
```

Reply:
Do note that it may not be in the best interest to put words in the mouth of the player character.

Reply:
> Do note that it may not be in the best interest to put words in the mouth of the player character.
Could you please elaborate on this?

Reply:
Previous capitalization is correct, by the way. 
<img width="816" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/3a03af5d-2f8e-43df-9ec3-38ac1ce7b2ee">

Reply:
> Previous capitalization is correct, by the way. <img alt="image" width="816" src="https://private-user-images.githubusercontent.com/98854112/288974116-3a03af5d-2f8e-43df-9ec3-38ac1ce7b2ee.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIxMjQ3MDIsIm5iZiI6MTcwMjEyNDQwMiwicGF0aCI6Ii85ODg1NDExMi8yODg5NzQxMTYtM2EwM2FmNWQtMmY4ZS00M2RmLTllYzMtMzhhYzFjZTdiMmVlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA5VDEyMjAwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBkNmJmN2NmMDQ4NTM0OGMwODU1ZjY1ZWJiODdkMTY2NTJkMGE4YjgwN2VlMDRjNDAwNWY4M2Y4ZWExNzVhMzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jl982c59HLASnmF9m3HAY8uSUZ3oDBPZjkZHyU7vSWU">

Whoops, that's my bad.

Reply:
> > Do note that it may not be in the best interest to put words in the mouth of the player character.
> 
> Could you please elaborate on this?

As for this, we generally don't like having the player character do or say something without the player's direction. We do this by allowing the Player to choose the option to say it themselves through either a choice, or a singular choice.

Stil, technically, it's not wrong to have it be a part of the overall conversation.

Reply:
That's sometimes (but not always) true, but it wouldn't make sense to insert a choice at this point in the conversation. It's meant to be an interruption.